_posts/2022-12-06-integrating-auth0-with-react.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+font: serif
+title: Integrating Auth0 Authentication into React
+date: 2022-12-06
```

Comment:
Note, you may want to update this prior to publish.  It must be a data in the past at the point of merge though